### PR TITLE
Replace unneeded inputs with new /ess dump in issue template

### PR DESCRIPTION
**Pull request by JRoy**

### CODE REVIEW

**Review by mdcfe — DISMISSED** [review]
Approved pending 2.19 release - would like to wait until `/ess dump` lands in a release to switch issue templates over.

File: .github/ISSUE_TEMPLATE/report-a-bug.yml
Comment (Chew) [inline .github/ISSUE_TEMPLATE/report-a-bug.yml:27]: Keep in mind markdown doesn't work in labels, see https://github.com/EssentialsX/Essentials/blob/b9309ac81ab81d44d7ea94c0b2ff4da593a83865/.github/ISSUE_TEMPLATE/report-a-bug.yml

File: .github/ISSUE_TEMPLATE/report-a-bug.yml
Comment (mdcfe) [inline .github/ISSUE_TEMPLATE/report-a-bug.yml]:
```suggestion
    label: "`/ess dump *` output"
    description: Run `/ess dump *` in the console, then copy and paste the link from the output of the command into this box. (If this command doesn't work, you should [update EssentialsX](https://essentialsx.net/downloads.html) and try again.)
```

**Review by mdcfe — APPROVED** [review]

**Pull request merged**
